### PR TITLE
add bicep support

### PR DESCRIPTION
**Pull request by markjbrown**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-cosmosdb-sql-rbac/azuredeploy.json [hunk 1]
```diff
@@ -67,43 +75,41 @@
...
+      "dependsOn": [
+        "[resourceId('Microsoft.DocumentDB/databaseAccounts', variables('accountName_var'))]"
+      ]
+    },
+    {
+      "type": "Microsoft.DocumentDB/databaseAccounts/sqlRoleAssignments",
+      "apiVersion": "2021-03-01-preview",
+      "name": "[format('{0}/{1}', variables('accountName_var'), variables('roleAssignmentId'))]",
+      "properties": {
+        "roleDefinitionId": "[resourceId('Microsoft.DocumentDB/databaseAccounts/sqlRoleDefinitions', split(format('{0}/{1}', variables('accountName_var'), variables('roleDefinitionId')), '/')[0], split(format('{0}/{1}', variables('accountName_var'), variables('roleDefinitionId')), '/')[1])]",
```
Comment (henrypbriffel) [inline 101-cosmosdb-sql-rbac/azuredeploy.json:105]: Henrypbriffel

**Review by henrypbriffel — APPROVED** [review]
May be a few things I missed but thorough.

**Review by henrypbriffel — APPROVED** [review]

**Comment by StephenWeatherford** [comment]
@bmoore-msft @markjbrown That's weird - I even deleted the offending build and it still shows up.

**Comment by markjbrown** [comment]
@StephenWeatherford  Where do I get the bicep badge?

**Comment by StephenWeatherford** [comment]
@markjbrown See this error: https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=60201&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=8ebc9149-1226-5c43-6fb4-06b80a585d5d&l=28

**Pull request closed without merging**